local_modules/configuration/views/wizard.pug

```diff
@@ -0,0 +1,52 @@
+extends /layout
+block head
+    include /datatables_css
+
+block content
+    .row
+        .page-header
+            h1 Node Configuration Wizard
+    .row
+        .panel.panel-default
+            .panel-heading Deployment Configuration
+            .panel-body
+                label #MOBS
+                input(type='text')
```

Comment:
Let's use the pub-bootstrap forms syntax:
http://rajasegar.github.io/JADE-Bootstrap/components.html#forms
` +input("text","txtMOBS","#MOBS","MOBS")`

Or the pure-bootstrap syntax:
```
form.form-signin(action="/login", method="post")
	h2.form-signin-heading Please sign in
	input.input-block-level(type="text", name="username", placeholder="Email address")
	input.input-block-level(type="password", name="password", placeholder="Password")
	label.checkbox.
		<input type="checkbox" value="remember-me" /> Remember me
	button.btn.btn-large.btn-primary(type="submit") Sign in
```